startup.sh

```diff
@@ -1,12 +1,4 @@
 #!/bin/sh
-
-git -C . fetch || git clone https://github.com/vladmandic/automatic.git .
-
-if [ ! -d /deps/venv ]; then
-    echo "Creating venv with system packages (torch, ipex, etc)"
-    python -m venv /deps/venv --system-site-packages
-fi
-
-export PATH=/deps/venv/bin:$PATH
-
-python ./launch.py "$@"
+git -C automatic fetch || git clone https://github.com/vladmandic/automatic.git
+cd automatic
+./webui.sh --use-ipex --listen
```

Comment:
We still need to accept variadic arguments here, to allow users to run the container with additional options (e.g. `--debug`, `--lowvram`) easily by [appending arguments to `docker run ...`](https://docs.docker.com/engine/reference/builder/#cmd)